helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -567,6 +567,8 @@ exporters:
   splunk_hec/o11y:
     endpoint: {{ include "splunk-otel-collector.o11yIngestUrl" . }}/v1/log
```

Comment:
Unrelated, but do you know why we don't use a hec receiver in the gateway? Is it because there's no access token passthrough?

Reply:
Do you mean using hec between agent and gateway? We use OTLP for performance reasons